docs/Makefile

```
@@ -10,7 +10,7 @@ BUILDDIR      = _build
 
 
 APIDOC        = sphinx-apidoc
-APIDOC_OPTS   = --implicit-namespaces --separate --module-first
+APIDOC_OPTS   = --implicit-namespaces --separate --module-first -t apidoc
```

```suggestion
APIDOC_OPTS   = --implicit-namespaces --separate --module-first --templatedir apidoc
```

We also need to update the sphinx version to support this.